src/groups/mqb/mqbstat/mqbstat_domainstats.h

```diff
@@ -72,6 +73,16 @@ class DomainStats {
         enum Enum { e_CFG_MSGS, e_CFG_BYTES, e_QUEUE_COUNT };
     };
 
+    //------------------------
+    // struct DomainStatsIndex
+    //------------------------
+
+    /// Namespace for the constants of stat values that applies to the queues
+    /// from the clients
+    struct DomainStatsIndex {
+        enum Enum { e_STAT_CFG_MSGS, e_STAT_CFG_BYTES, e_STAT_QUEUE_COUNT };
+    };
```

Comment:
It's good that the enum belongs to this class now, but it's better to move it to private types